CONTRIBUTING.md

```diff
@@ -99,11 +99,11 @@ submit them to the base repository using a pull request.
     
     Where `<branch>` is the branch name you used in step 2.
 
-    If asked for github username and password, make sure to use your [Personal
-    Access Token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line) instead of your password, since two-factor authentication is
-    enabled by default.
+    Note (Googlers-only): If asked for github username and password, make sure
```

Comment:
Sorry to be a nag about this, but instead of (Googlers-only) I'd do something like:

```
**Note**: If you have two factor authentication enabled, make sure to use your [Personal Access Token](link) instead of your password. 
```

That way we also account for the case where someone has 2fa enabled but happens to belong to a different org or just has it set up for personal use.

Reply:
Done!